### DISCUSSION TITLE
What is the difference between wildcard and normal domain registration?

### DISCUSSION THREAD

**Discussion by j4nos** [vercel/community#734] [Help] [answered]
Is this document still relevant? It is from 2019. https://vercel.com/blog/wildcard-domains

Anyway, it says the difference between wildcard  and non-wildcard  is in step 3. Either I set `*.static.fun` or `static.fun`, right?

1. Why do I not see any difference in DNS records?
2. Is it normal, if with `*.` I got `DEPLOYMENT_NOT_FOUND` for each route? Or it takes some time, to get the changes applied, 2d?
3. Do I have to use Vercel REST Api to update DNS records with `POST /v2/domains/{domain}/records`?

**Comment by amyegan** [comment, marked as answer]
That document is using old nameservers, but is generally still relevant. We have the current process for [setting up wildcard domains](https://vercel.com/docs/concepts/projects/custom-domains#wildcard-domains) outlined in our docs.

The DEPLOYMENT_NOT_FOUND error means that your production deployment could not be located. This could happen if the domain is not assigned to a project or you do not have a production deployment.

If you just added the domain, it could take a little while for the DNS records to update. It does not normally take a very long time, but this depends on a few different factors. It typically only takes a few minutes in my experience. You should see that the domain has "valid configuration" in your Project's domain settings when it's ready to use.
<img width="413" alt="image" src="https://user-images.githubusercontent.com/8247105/179768396-c062f40c-0c08-454e-85f7-1d85a71cf26f.png">

There are multiple options for updating DNS records. You can update DNS records manually from the Dashboard. The Vercel CLI has options to manage [domains](https://vercel.com/docs/cli#commands/domains) or set a [deployment alias](https://vercel.com/docs/cli#commands/alias). And you already know about the [API endpoints for domains](https://vercel.com/docs/rest-api#endpoints/domains)
